test/models/members/project_member_test.rb

```diff
@@ -126,4 +126,11 @@ def setup
       Member.restore(@project_member.id, recursive: true)
     end
   end
+
+  test '#scope for_namespace_and_ancestors returns the correct collection' do
```

Comment:
**nitpick (blocking):** Can you update this test to include testing for members from the Group and Ancestors that the project belongs to. So similar to NamespaceGroupLink and also make sure that the total is the same.

Reply:
Updated in 07d46704b0aab5511c116e8688c9ae977526cd96